### PR TITLE
COMMERCE-2469 use the correct groupId when getting virtualOrderItems

### PR DESCRIPTION
**Pull request by luca-pellizzon**
@marco-leo @ethanbustad

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test

File: commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java [hunk 1]
```diff
@@ -304,15 +304,18 @@ public PortletURL getPortletURL() throws PortalException {
 				getLiferayPortletRequest(),
 			getPortletURL(), null, "no-items-were-found");
 
+		long commerceChannelGroupIdBySiteGroupId =
```
Comment (marco-leo) [inline commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java]: @luca-pellizzon  name should be commerceChannelGroupId

File: commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java [hunk 2]
```diff
@@ -304,15 +304,18 @@ public PortletURL getPortletURL() throws PortalException {
...
+			_commerceChannelLocalService.getCommerceChannelGroupIdBySiteGroupId(
+				_commerceVirtualOrderItemContentRequestHelper.getSiteGroupId());
```
Comment (marco-leo) [inline commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java]: getScopeGroupId  instead 0f getSiteGroupId

**Review by marco-leo — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
@marco-leo updated

**Comment by marco-leo** [comment]
ci:test

File: commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java [hunk 3]
```diff
@@ -304,16 +304,19 @@ public PortletURL getPortletURL() throws PortalException {
...
+				_commerceVirtualOrderItemContentRequestHelper.
+					getScopeGroupId());
+
 		int total =
 			_commerceVirtualOrderItemLocalService.
 				getCommerceVirtualOrderItemsCount(
-					_commerceVirtualOrderItemContentRequestHelper.
-						getScopeGroupId(),
+					commerceChannelGroupId,
 					_commerceVirtualOrderItemContentRequestHelper.getUserId());
```
Comment (marco-leo) [inline commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java]: @luca-pellizzon I suppose this is an error and need to pass accountId no userId

**Review by marco-leo — CHANGES_REQUESTED** [review]

File: commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java (hunk 3 above)
Comment (luca-pellizzon) [reply commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java]: didn't notice it, but yes it does need accountId

**Comment by luca-pellizzon** [comment]
@marco-leo  updated

**Pull request merged**